idris-commands.el

```diff
@@ -238,7 +238,11 @@ line."
            (lambda (_condition)
              (when (member 'warnings-tree idris-warnings-printing)
                (idris-list-compiler-notes)
-               (pop-to-buffer (idris-buffer-name :notes)))))))
+	       ( if idris-stay-in-current-window-on-compiler-error
+                 (display-buffer (idris-buffer-name :notes))
+                 (pop-to-buffer (idris-buffer-name :notes))
+	       )
+	       )))))
```

Comment:
thx - but please follow the emacs lisp syntax rules used in this repository, namely: no whitespace before `if`, no newline before closing parens...